src/deployment/default/fonts.ts

```diff
@@ -1,6 +1,12 @@
+/**
```

Comment:
This now matches the Open Source Python one (Helvetica Now is out commercial font on .org).